doc/links.txt

```diff
@@ -73,3 +73,5 @@
 .. _`NMEA report sample`: http://aprs.gids.nl/nmea/#gsa
 
 .. _`Bluetooth Core Specification`: https://www.bluetooth.com/specifications/specs/core-specification/
+
+.. _`Appropriate Language Mapping Table`: https://btprodspecificationrefs.blob.core.windows.net/language-mapping/Appropriate_Language_Mapping_Table.pdf
```

Comment:
Would be better to use 'official' link from the Bluetooth spec. Less probable it will change.
```suggestion
.. _`Appropriate Language Mapping Table`: https://www.bluetooth.com/language-mapping/Appropriate-Language-Mapping-Table
```